src/test/java/com/code_intelligence/jazzer/mutation/mutator/lang/ByteArrayMutatorTest.java

```diff
@@ -0,0 +1,143 @@
+  }
+
+  @Test
+  void testBasicFunction() {
+    SerializingMutator<byte[]> mutator =
+        (SerializingMutator<byte[]>) LangMutators.newFactory().createOrThrow(
+            new TypeHolder<byte[]>() {}.annotatedType());
+    assertThat(mutator.toString()).isEqualTo("Nullable<byte[]>");
+
+    byte[] arr;
+    try (MockPseudoRandom prng = mockPseudoRandom(false, 5, new byte[] {1, 2, 3, 4, 5})) {
+      arr = mutator.init(prng);
+    }
+    assertThat(arr).isEqualTo(new byte[] {1, 2, 3, 4, 5});
+
+    try (MockPseudoRandom prng = mockPseudoRandom(false)) {
+      arr = mutator.mutate(arr, prng);
+    }
+    assertThat(arr).isEqualTo(new byte[] {2, 4, 6, 8, 10, 6, 7, 8, 9});
+  }
+
+  @Test
+  void testMaxLength() {
+    SerializingMutator<byte[]> mutator =
+        (SerializingMutator<byte[]>) LangMutators.newFactory().createOrThrow(
+            new TypeHolder<byte @NotNull @WithLength(max = 10)[]>() {}.annotatedType());
+    assertThat(mutator.toString()).isEqualTo("byte[]");
+
+    byte[] arr;
+    try (MockPseudoRandom prng = mockPseudoRandom(8, new byte[] {1, 2, 3, 4, 5, 6, 7, 8})) {
+      arr = mutator.init(prng);
+    }
+    assertThat(arr).isEqualTo(new byte[] {1, 2, 3, 4, 5, 6, 7, 8});
+
+    try (MockPseudoRandom prng = mockPseudoRandom()) {
+      arr = mutator.mutate(arr, prng);
+    }
+    assertThat(arr).hasLength(10);
+    assertThat(arr).isEqualTo(new byte[] {2, 4, 6, 8, 10, 12, 14, 16, 9, 10});
+
+    try (MockPseudoRandom prng = mockPseudoRandom()) {
+      arr = mutator.mutate(arr, prng);
+    }
+    assertThat(arr).hasLength(10);
+    assertThat(arr).isEqualTo(new byte[] {3, 6, 9, 12, 15, 18, 21, 24, 18, 20});
+  }
+
+  @Test
+  void testMaxLengthInitClamp() {
+    SerializingMutator<byte[]> mutator =
+        (SerializingMutator<byte[]>) LangMutators.newFactory().createOrThrow(
+            new TypeHolder<byte @NotNull @WithLength(max = 5)[]>() {}.annotatedType());
+    assertThat(mutator.toString()).isEqualTo("byte[]");
+
+    // init will call closedrange(min, max) and the mock prng will assert that the given value
+    // above is between those values which we want to fail here to show that we're properly
+    // clamping the range
+    Assertions.assertThrows(AssertionError.class, () -> {
+      byte[] arr;
+      try (MockPseudoRandom prng = mockPseudoRandom(10)) {
+        arr = mutator.init(prng);
+      }
+    });
+  }
+
+  @Test
+  void testMinLengthInitClamp() {
+    SerializingMutator<byte[]> mutator =
+        (SerializingMutator<byte[]>) LangMutators.newFactory().createOrThrow(
+            new TypeHolder<byte @NotNull @WithLength(min = 5)[]>() {}.annotatedType());
+    assertThat(mutator.toString()).isEqualTo("byte[]");
+
+    // init will call closedrange(min, max) and the mock prng will assert that the given value
+    // above is between those values which we want to fail here to show that we're properly
+    // clamping the range
+    Assertions.assertThrows(AssertionError.class, () -> {
```

Comment:
Could you move the `assertThrows` inside the try block? Otherwise it could mask unexpected `AssertionError`s coming from the try block calling `MockPseudoRandom#close`. Similar consideration applies to the try block in the test above.